solidity/random-beacon/hardhat.config.ts

```diff
@@ -37,8 +38,11 @@ const config: HardhatUserConfig = {
           ? parseInt(process.env.FORKING_BLOCK, 10)
           : undefined,
       },
-      accounts: { count: 70 },
+      // We want to have 10 accounts for various tests and `5 * 64` accounts to use
+      // unique addresses in staking for each operator.
+      accounts: { count: 10 + 5 * 64 },
       tags: ["local"],
+      allowUnlimitedContractSize: true,
```

Comment:
This is only for `RandomBeaconStub`, right? It is quite dangerous because we may end up with non-deployable `RandomBeacon` and not even notice it. Would it be possible to branch-off from this PR, remove this line, and try to make tests passing by, for example, dividing `RandomBeaconStub` into several per-test stubs?

Reply:
Yeah, the `RandomBeaconStub` is problematic. I already tried to reduce the size of the stub in https://github.com/keep-network/keep-core/pull/2906. There is not much room for `RandomBeacon` to grow as it's `24.091 KB` already.

As I mentioned in the commit https://github.com/keep-network/keep-core/commit/90e659419be9cff8fd9bf5ccf26509fe132c6dbf we're protected by the `hardhat-contract-sizer`. Whenever we run `yarn build` the plugin will verify the size of the `RandomBeacon`.

> Would it be possible to branch-off from this PR, remove this line, and try to make tests passing by, for example, dividing RandomBeaconStub into several per-test stubs?

If the contract sizer is not enough in your opinion, I can try slicing the contract.

Reply:
Let's try slicing and we'll see where we end up with. I am not a fan of stubs adding a lot of functionality because in the end, we do not know what we are testing - the final contract or stub. On the other hand, having just one stub *could* (unsure) be better for aggregating gas usage reports by hardhat plugin.

Reply:
FYI `hardhat-contract-sizer` will fail contracts build in case of a too big contract. It means that CI will fail and PR won't be mergeable. See example PR https://github.com/keep-network/keep-core/pull/2912